contracts/Strategy.sol

```diff
@@ -812,12 +819,13 @@ contract Strategy is BaseStrategy, DydxFlashloanBase, ICallee {
     }
 
     function ethToWant(uint256 _amtInWei) public view override returns (uint256) {
-        return priceCheck(weth, address(want), _amtInWei);
+        return _amtInWei;
     }
 
     function liquidateAllPositions() internal override returns (uint256 _amountFreed) {
         (_amountFreed,) = liquidatePosition(vault.debtOutstanding());
         (uint256 deposits, uint256 borrows) = getCurrentPosition();
```

Comment:
does it make sense to put this and position calc into the !forceMigrate if clause?

Reply:
I dont think so? Explain more what you mean?

Reply:
ahh you are right:
fe028c8eea2e426bf9df7e11c8035027430abf76